bip-0048.mediawiki

```diff
@@ -240,6 +243,13 @@ Public derivation is used at this level.
 |change
 |second
 |m / 48' / 1' / 1' / 2' / 1 / 1
+|-
+|testnet
+|first
+|p2tr
+|external
+|first
+|m / 48' / 1' / 0' / 3' / 0 / 0
```

Comment:
@ben-kaufman @Fonta1n3 Should there be mainnet p2tr entry as well?

Reply:
I didn't add because there was none for p2sh-p2wsh, only testnet, but we could add for both I think.

Reply:
> I didn't add because there was none for p2sh-p2wsh, only testnet, but we could add for both I think.

Would you like to make a PR?

Reply:
Sure https://github.com/bitcoin/bips/pull/1846